apps/webapp/utils/format-number.ts

```diff
@@ -1,11 +1,22 @@
 //  This function takes a number and formats it for display with exactly two decimal places.
 // If the number is an integer or has fewer than two decimal places, it will be displayed with exactly two decimal places. For example, if number is 2000, it will be formatted as "2,000.00".
 // If the number is a floating-point number with more than two decimal places, it will be rounded to exactly two decimal places. For example, if number is 2001.1, it will be formatted as "2,001.10" because it is rounded to two decimal places.
-export const formatNumber = (number: number): string =>
+export const formatNumber = (number: number): string => {
```

Comment:
If there aren't any "cents" it should not show anything. For example:

Right now we have `8,372.00`. But given there aren't cents, we should just have `8,372`